extensions/amp-vidazoo-widget/0.1/test/validator-amp-vidazoo-widget.html

```diff
@@ -0,0 +1,106 @@
+<!--
+  Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS-IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the license.
+-->
+<!--
+  Test Description:
+  Tests for the amp-vidazoo-widget tag.
+-->
+<!doctype html>
+<html ⚡>
+
+<head>
+    <meta charset="utf-8">
+    <link rel="canonical" href="./regular-html-version.html">
+    <meta name="viewport" content="width=device-width">
+    <style amp-boilerplate>
```

Comment:
For succinctness, can you squeeze all the amp-boilerplate CSS onto one line, as it is done at https://validator.amp.dev/ and in other validator tests? It obscures the useful part of this test, which is in the `<body>`.